ckanext/validation/helpers.py

```diff
@@ -32,22 +32,38 @@ def get_validation_badge(resource, in_listing=False):
     else:
         status = 'unknown'
 
-    validation_url = url_for(
+    tags = 'href="' + url_for(
```

Comment:
this change looks weird.
1. Variable name is kind of misleading (`validation_url` sounded better IMO)
2. Why put `href=` as part of the variable and remove it from line `54`?